core/ide/src/main/java/org/alice/ide/codedrop/CodePanelWithDropReceptor.java

```diff
@@ -335,120 +335,96 @@ public final BlockStatementIndexPair dragUpdated(DragStep step) {
 
     @Override
     protected Triggerable dragDroppedPostRejectorCheck(DragStep step) {
-      Triggerable rv = null;
       final DragModel dragModel = step.getModel();
-      DragComponent dragSource = step.getDragSource();
       final MouseEvent eSource = step.getLatestMouseEvent();
       final StatementListPropertyView statementListPropertyPane = this.currentUnder;
```

Comment:
the new code is so much more readable.  the dif is not helpful, so I did my best, but at least if this did accidentally break something it will be easier to figure out